clients/bigquery/storagewrite_test.go

```diff
@@ -129,9 +129,9 @@ func TestRowToMessage(t *testing.T) {
 		"c_float_int32":    int32(1234),
 		"c_float_int64":    int64(1234),
 		"c_float_string":   "4444.55555",
-		"c_numeric":        decimal.NewDecimal(nil, 5, big.NewFloat(3.1415926)),
+		"c_numeric":        decimal.NewDecimal(nil, numbers.MustParseDecimal("3.14159")),
```

Comment:
We now rely on the exponent of the decimal instead of passing in a scale, so I the decimal places here were trimmed to 5.